src/apps/fee-list/stackable-fees/fee-details-content.tsx

```diff
@@ -34,78 +35,66 @@ const FeeDetailsContent: FC<FeeDetailsContentProps> = ({ feeDetailsData }) => {
-          <CheckBox
-            id="checkbox_id__fee_details"
-            onChecked={() => handleAcceptedTerms()}
-            label={
-              <>
-                {t("iAcceptText")}{" "}
-                <Link href={termsOfTradeUrl}>
-                  {t("termsOfTradeText")}
-                  <sup>*</sup>
-                </Link>
-              </>
-            }
+    <div className="modal-loan__container">
+      <div className="modal-loan__header">
+        <div className="mr-32">
+          <FeeStatusCircle
+            dueDate={dueDate || ""}
+            feeCreationDate={creationDate}
           />
-          <div>
-            <p>{amount},-</p>
-          </div>
-          {!showPaymentButton && (
-            <button
-              type="button"
-              className="btn-primary btn-outline btn-small arrow__hover--right-small"
-              disabled={!showPaymentButton}
-            >
-              {t("payText")}
-            </button>
-          )}
-          {showPaymentButton && (
-            <Link
-              className="btn-primary btn-filled btn-small arrow__hover--right-small disabled"
-              href={new URL(paymentOverviewUrl)}
-            >
-              {t("payText")}
-            </Link>
-          )}
-
-          {/* )} */}
         </div>
-        <ul className="modal-loan__list-container">
-          <li className="modal-loan__list">
-            <ul className="modal-loan__list-materials">
-              {materials.map(({ recordId }) => (
-                <StackableFeesList
-                  key={recordId}
-                  faust={`${recordId}` as FaustId}
-                  creationDateFormatted={creationDateFormatted}
-                />
-              ))}
-            </ul>
-          </li>
-        </ul>
+        <div>
+          <h2 className="modal-loan__title text-header-h2">
+            {t("turnedInText", {
+              placeholders: { "@date": creationDateFormatted }
+            })}
+          </h2>
+        </div>
       </div>
+      <GroupModalContent
+        amountOfSelectableMaterials={0}
+        selectableMaterials={[]}
+        selectedMaterials={[]}
+        buttonComponent={
+          <>
+            <AcceptTermsCheckbox handleAcceptedTerms={handleAcceptedTerms} />
+            <div>
+              <p>
+                {t("amountText", {
+                  placeholders: { "@amount": amount }
+                })}
+              </p>
+            </div>
+            {!showPaymentButton && (
+              <button
+                type="button"
+                className="btn-primary btn-outline btn-small arrow__hover--right-small"
+                disabled={!showPaymentButton}
+              >
+                {t("payText")}
+              </button>
+            )}
+            {showPaymentButton && (
+              <Link
+                className="btn-primary btn-filled btn-small arrow__hover--right-small disabled"
+                href={new URL(paymentOverviewUrl)}
```

Comment:
If the url comes from the useUrl hook you do not need to create a new url object

Reply:
done